ESlint failing to start due to React dependency?

I've done the walk-thru multiple times (including the part where I have to *un*install everything and start over again.

ESlint isn't doing *anything* to re-format malformed lines as shown in the video.  When I click on the 'ESlint' icon in the toolbar (which has to be enabled separately now, apparently) I get the lines shown below:

```
[Info  - 12:14:59 AM] ESLint server running in node v12.8.1
[Info  - 12:14:59 AM] ESLint server is running.
[Info  - 12:17:16 AM] ESLint library loaded from: C:\Users\memil\Downloads\Beginner Javascript\course files\node_modules\eslint\lib\api.js
Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.
[Info  - 12:17:18 AM] Error while loading rule 'react/jsx-no-bind': Cannot find module 'react' from 'C:\Users\memil\Downloads\Beginner Javascript\course files' Occurred while linting C:\Users\memil\Downloads\Beginner Javascript\course files\playground\some-FINISHED.js
[Info  - 12:21:19 AM] Error while loading rule 'react/display-name': Cannot find module 'react' from 'C:\Users\memil\Downloads\Beginner Javascript\course files' Occurred while linting C:\Users\memil\Downloads\Beginner Javascript\course files\playground\some-FINISHED.js
[Info  - 12:21:40 AM] Error while loading rule 'react/display-name': Cannot find module 'react' from 'C:\Users\memil\Downloads\Beginner Javascript\course files' Occurred while linting C:\Users\memil\Downloads\Beginner Javascript\course files\playground\some-FINISHED.js

```

After running `npm install react`,  I get a success message, but a number of other warnings about peerdeps (sorry, didn't capture those).

Anway, after installing react, which is an odd dependency since the Beginner Javascript course doesn't - as far as I can tell - use react or even touch on it, now I have ESlint about half-way working like shown in the video (#5).

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "eslint-plugin-import": "^2.18.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.0.1",
-    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react": "^7.20.6",
     "eslint-plugin-react-hooks": "^1.4.0",
     "prettier": "^1.16.4"
   },
@@ -36,7 +36,7 @@
     "eslint-plugin-import": "^2.18.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.0.1",
-    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react": "^7.20.6",
     "eslint-plugin-react-hooks": "^1.4.0",
     "prettier": "^1.16.4"
   },

